pytest_axe/pytest_axe.py

```diff
@@ -59,54 +70,76 @@ def report(self, violations):
         :return report: Readable report of violations.
         :rtype: string
         """
-        string = ''
-        string += 'Found ' + str(len(violations)) + ' accessibility violations:'
+        string = ""
+        string += "Found " + str(len(violations)) + " accessibility violations:"
```

Comment:
Can you change this to `string = "Found " + str(len(violations)) + " accessibility violations:"`

A nit, but not requirement, change `string` to something like `violations` to give the variable name a bit of meaning.